docs/source/pillars/computing_technology.md

```diff
@@ -59,39 +59,67 @@ _The tools used by researchers inside a TRE, such as programming languages, IDEs
 * - Statement
   - Guidance
   - Importance
-* - Your TRE must provide software applications that are relevant to working with the data in the TRE.
-  - The tools provided will depend on the types of data in the TRE, and the expectations of users of the TRE.
-    For users working in a TRE via a virtual desktop, this may include programming languages such as Python and R, integrated development environments, Jupyter notebooks, office type applications such as word processors and spreadsheets, command line tools, etc.
-    TREs with non-desktop interfaces should similarly consider carefully which applications are best suited for the researchers needs when interacting with the data, for example "point and click" GUI tools for querying a database and generating plots of data.
-    The set of tools should be reviewed regularly to ensure they are up to date.
-  - Mandatory
+* - Your TRE should be accessed via a user interface accessible using commonly available applications.
+  - TREs which allow users to connect from their own devices should not require the installation of any bespoke TRE application on the user's device.
+    In practice a web browser is the most common way to achieve this.
+  - Recommended
 * - Your TRE must provide clear guidance on how to use software tools and work with data in the TRE.
   - TREs that provide a virtual desktop environment for researchers to work in should provide documentation detailing the available tools.
     TREs where the analysis code is developed on the access machine (as opppose to within the TRE) should provide documentation detailing the mechanism by which code is submitted to the TRE.
   - Mandatory
-* - Your TRE should provide tools to encourage best-practice in reproducibly analysing data.
-  - Reproducibility of analyses improves auditability and accountability of how data has been used, as well as being best-practice in research.
-    This may include version control software, and tools for developing and running data analysis pipelines.
-  - Recommended
 * - Your TRE should, where possible, automatically apply security related updates for user software.
   - Reducing the risk of exploitable vulnerabilities in installed software will increase the security of your TRE.
   - Recommended
 * - Your TRE could provide shared services that are accessible to users in the same project.
   - This may include shared file storage, databases, collaborative writing, and other web applications.
     This must only be shared amongst users within the same project.
   - Optional
+* - Your TRE could include licenced commercial software if required by researchers, but additional risks must be recorded and mitigated where neccesary.
+  - For example, if an application must connect to an external licensing server, you must be confident that only licensing information is sent to this server, and that any network connections are secure.
+  - Optional
+```
+
+### Code Version Control System
```

Comment:
```suggestion
### Enabling Reproducible Data Analysis and Version Control of Research Code
```

Reply:
I don't love the current name but I think the point is that it's an "application" in architecture-language so the name needs to be a description of an object/system.

Reply:
How about that? Feel free to reject entirely

Reply:
It's not wrong, but it's a bit verbose! :joy:

Reply:
how about now?

Reply:
I'm going to ignore this to stay in-sync with WP3, but please comment on the doc in teams or discuss with @machintim when he gets back.